packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -532,6 +544,18 @@ export class StatsAnalyzer extends EventsScope {
             `StatsAnalyzer:index#compareLastStatsResult --> No audio RTP packets sent`,
             currentStats.totalPacketsSent
           );
+          if (this.lastEmittedStartStopEvent.audio.local === EVENTS.LOCAL_MEDIA_STARTED) {
```

Comment:
the first time this gets called, this condition is checked before `emitStartStopEvents()` is called, so `lastEmittedStartStopEvent.audio.local` will always be undefined the first time and so we will only emit the event the second time this function is called - I think that's 5 seconds later. Is that expected? It doesn't feel right.

Reply:
actually I misread the code I thought this was checking for "stopped" event, but I see it's checking for "started" - I guess this is to prevent the event firing more than once, so that's OK, it's a bit confusing but it will work, except for the case when we don't send anything from the start, then I think lastEmittedStartStopEvent remains undefined (it's never set to STARTED if we never send anything), so we won't trigger the event, am I right?

Reply:
Yes. This is to prevent sending metric to amplitude multiple times. And if we dont send anything at the start the lastEmittedEvent will be undefined so we wont trigger any event.

Reply:
> And if we dont send anything at the start the lastEmittedEvent will be undefined so we wont trigger any event.

but we should send the NO_AUDIO_SENT event in that case, shouldn't we?